alns/accept/tests/test_late_acceptance_hill_climbing.py

```diff
@@ -0,0 +1,116 @@
+import numpy.random as rnd
+from numpy.testing import assert_, assert_equal, assert_raises
+from pytest import mark
+
+from alns.accept import LateAcceptanceHillClimbing
+from alns.tests.states import Zero, One, Two
+
+
+@mark.parametrize("history_length", [-0.01, -10, 1.5])
+def test_raises_invalid_history_length(history_length):
+    with assert_raises(ValueError):
+        LateAcceptanceHillClimbing(history_length=history_length)
+
+
+@mark.parametrize(
+    "history_length, greedy, collect_better",
+    [
+        (1, True, True),
+        (10, False, True),
+        (100, True, False),
+        (1000, False, False),
+    ],
+)
+def test_properties(history_length, greedy, collect_better):
+    lahc = LateAcceptanceHillClimbing(history_length, greedy, collect_better)
+
+    assert_equal(lahc.history_length, history_length)
+    assert_equal(lahc.collect_better, collect_better)
+    assert_equal(lahc.greedy, greedy)
```

Comment:
In light of the issue, please also test `lookback_period = 0`. I am unsure in which of these two tests that should go, depending on whether we accept zero as a valid value or not :-).

Reply:
I think we should allow `lookback_period=0`, it's also in the original implementation. Tested in new version.